integ_entrypoint.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
nit: Should this be under `.scripts/`?

Reply:
I didn't put it under `scripts` because those are generally project level scripts and this is really integration test code. I put it in the root to live next to the Dockerfile.

Personally, I think we should move both the Dockerfile and setup script into `integration` somewhere.